nilearn/input_data/multi_nifti_masker.py

```diff
@@ -178,9 +178,7 @@ def fit(self, imgs=None, y=None):
                 raise ValueError("Unknown value of mask_strategy '%s'. "
                     "Acceptable values are 'background' and 'epi'.")
 
-            self.mask_img_ = self._cache(
-                        compute_mask,
-                        func_memory_level=1,
+            self.mask_img_ = self._cache(compute_mask, 1,
```

Comment:
I would prefer, for readability, still having the name of the keyword argument in the code. It serves to explain what the '1' means.